docs/index.md

```diff
@@ -53,27 +53,27 @@ outputs:
 So *pydantic* uses some cool new language features, but why should I actually go and use it?
 
 **no brainfuck**
-: no new schema definition micro-language to learn. If you know python (and perhaps skim the
+: there's no new schema definition micro-language to learn. If you know python (and perhaps skim the
   [type hinting docs](https://docs.python.org/3/library/typing.html)) you know how to use *pydantic*.
 
 **plays nicely with your IDE/linter/brain**
-: because *pydantic* data structures are just instances of classes you define; auto-completion, linting,
-  [mypy](usage/mypy.md), IDEs (especially [PyCharm](pycharm_plugin.md)) and your intuition should 
+: *pydantic* data structures are just instances of classes you define, so auto-completion, linting,
+  [mypy](usage/mypy.md), IDEs (especially [PyCharm](pycharm_plugin.md)), and your intuition should 
   all work properly with your validated data.
 
 **dual use**
-: *pydantic's* [BaseSettings](usage/settings.md) class allows it to be used in both a "validate this request data"
-  context and "load my system settings" context. The main difference being that system settings can have defaults
-  changed by environment variables and more complex objects like DSNs and python objects are often required.
+: *pydantic's* [BaseSettings](usage/settings.md) class allows *pydantic* to be used in both a "validate this request
+  data" context, and it a "load my system settings" context. The main difference is that system settings can read
+  defaults from environment variables, and more complex objects like DSNs and python objects are often required.
```

Comment:
after an extended discussion in the office we think there shouldn't be a comment here.

Reply:
I'm not sure whether you were referring to both commas or just the first one. Either way, I think you're right -- I tend to do commas by intuition based on how I am inclined to read the sentence aloud, but thinking more logically here I see they aren't necessary.

I'm inclined to keep the second comma though since I find it hard to parse the sentence otherwise. (It's not clear that the second part is distinct from the first until you reach the end; I find the comma helps me realize this as I go.)

Also, I think it should probably be changed from saying "The main difference is" to "The main differences are" given there are two 😄.

I'll rework it tonight.